src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -1662,8 +1662,11 @@ def export_coreml(self, filename,
         """
         options = {}
         options['include_non_maximum_suppression'] = include_non_maximum_suppression
-        options['confidence_threshold'] = confidence_threshold
-        options['iou_threshold'] = iou_threshold
+        options['version'] = self._get_version()
+        if confidence_threshold != None:
```

Comment:
Let's follow PEP 8 and replace `!=` with `is not`.

Reply:
good! ;)